src/pages/Txs/Txs.tsx

```diff
@@ -29,15 +30,28 @@ const getRow = (response: TxInfo, chainID: string, isClassic?: boolean) => {
         <TxAmount amount={fee.amount} denom={fee.denom} />
       )}
     </span>,
+    <span>{isClassic ? <TaxRateAmount logs={logs} /> : ""}</span>,
+    <span>
+      <Finder q="blocks" v={String(height)}>
+        {String(height)}
```

Comment:
height is already showing
please remove this span tag

Reply:
Done.